test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -399,6 +399,23 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
         $result = ExecuteWebCommand -command $command
         $result.Error | Should BeNullOrEmpty
     }
+
+    It "Validate StandardMethod and CustomMethod parameter sets" {
+        
+        #Validate that parameter sets are functioning correctly
+        $command = "Invoke-RestMethod -Uri 'http://sandbox.lee.io/method.php' -Method GET -CustomMethod 'TEST'"
+        $result = ExecuteWebCommand -command $command
+        $result.Error | Should Not BeNullOrEmpty
```

Comment:
You should use the `ShouldBeErrorId` pattern, see https://github.com/PowerShell/PowerShell/blob/master/test/powershell/engine/Module/TestModuleManifest.ps1#L50